_source/logzio_collections/_metrics-sources/kubernetes-over-helm.md

```diff
@@ -80,7 +80,8 @@ This method of deployment gives you greater control over the configuration.
 ##### Add the logzio-k8s-metrics repo to your helm repo list
```

Comment:
Is this content relevant for Prometheus metrics? 
**_source/logzio_collections/_metrics-sources/kubernetes-over-helm.md**  is in a tab that has been hidden from users.

Should the content be added to _**_source/logzio_collections/_prometheus-sources**_